packages/ui/src/components/tabnav.tsx

```diff
@@ -8,13 +8,16 @@ const TabNavRoot: React.FC = ({ children }) => {
   )
 }
 
-const commonClasses = 'flex h-full items-center text-center cursor-pointer'
+const commonClasses =
+  'flex h-full items-center text-center cursor-pointer border-solid px-0 font-normal text-foreground-2 duration-150 ease-in-out hover:text-foreground-1'
 
 const TabNavItem: React.FC<NavLinkProps> = ({ children, ...props }) => {
   return (
     <NavLink
       role="tab"
-      className={({ isActive }) => (isActive ? `${commonClasses} text-primary border-b border-primary` : commonClasses)}
+      className={({ isActive }) =>
+        isActive ? `${commonClasses} text-primary border-b-2 border-borders-9` : commonClasses
```

Comment:
since same classes are being added in common and active, we should use `cn` to combine them